src/Nethermind/Nethermind.Consensus/Processing/ProcessingStats.cs

```diff
@@ -18,11 +18,26 @@
 
 namespace Nethermind.Consensus.Processing
 {
+    public class BlockStatistics
+    {
+        public long BlockCount { get; internal set; }
+        public long BlockFrom { get; internal set; }
+        public long BlockTo { get; internal set; }
+        public double ProcessingMs { get; internal set; }
+        public double SlotMs { get; internal set; }
+        public double MgasPerSecond { get; internal set; }
+        public float MinGas { get; internal set; }
+        public float MedianGas { get; internal set; }
+        public float AveGas { get; internal set; }
```

Comment:
```suggestion
        public float MeanGas { get; internal set; }
```
Assuming that's what this means?

Reply:
not resolved?